lang/cs/Org.Apache.REEF.Network/Group/Driver/Impl/ContextManager.cs

```diff
@@ -0,0 +1,149 @@
+﻿// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Can you comments what might need to be in this class for IMRU specific stuff that would help to foresee the situation.